CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# Release 0.26.12
+
+## What's New
+
+- Ziti CLI
+  - Added CLI flags for setting router tunneler capability
+
+## Ziti CLI
+### Added CLI flags for setting router tunneler capability
+Ziti CLI `ziti create config router edge` now has two new flags; `--tunnelerMode` and `--lanInterface`
+#### --tunnelerMode
+The `--tunnelerMode` flag enables tunneling and sets the tunneler mode. Currently, there are `none`, `host` and `tproxy` 
+modes. The default tunneler mode is `host` mode, choosing `none` will disable tunnel capabilities for the router.
+
+Examples:
+```shell
+ziti create config router edge --routerName myRouter --tunnelerMode tproxy
+
+ziti create config router edge --routerName myRouter --tunnelerMode none
+```
+#### --lanInterface
+If using the `tproxy` tunneler mode, there is an optional `lanIf` section in the config to identify an interface to use.
```

Comment:
what happens with `tunnelerMode: none` and `lanInterface` is specfified?

Reply:
Nothing, the lanIf only shows up in the config if tproxy is set as the mode.

Reply:
Hmmm, actually I don't like what happened here, I'm going to update this. I don't like the blank lines commented out and I don't like that it says `mode: none`. Obviously they don't affect the outcome but it just seems, unrefined to me.

```
listeners:
# bindings of edge and tunnel requires an "edge" section below
  - binding: edge
    address: tls:0.0.0.0:3022
    options:
      advertise: MacBook-Pro.local:3022
      connectTimeoutMs: 1000
      getSessionTimeout: 60
#  - binding: tunnel
#    options:
#      mode: none #tproxy|host
#
#
```

Reply:
so are we waiting for a new round of changes? regardless I think these are fine to merge into the other branch